bundles/org.openhab.binding.ambientweather/pom.xml

```diff
@@ -42,7 +42,7 @@
     <dependency>
       <groupId>org.json</groupId>
       <artifactId>json</artifactId>
-      <version>20180813</version>
+      <version>20230227</version>
```

Comment:
Can you upload this dependency to Artifactory @kaikreuzer?

Reply:
Done.